yellow-paper/docs/public-vm/gen/_InstructionSet.mdx

```diff
@@ -362,7 +362,7 @@ Click on an instruction name to jump to its section.
 	<td><Markdown>Get the historical blocks tree root as of the specified block number.</Markdown></td>
```

Comment:
Naming in this file is stale. But I am not really sure what this file is doing  so maybe it's ok to ignore. (it's in gen folder so it could be just some generated thing)